rllib/agents/dqn/dqn.py

```diff
@@ -34,37 +35,30 @@
     # N-step Q learning
     "n_step": 1,
 
-    # === Exploration ===
-    # Max num timesteps for annealing schedules. Exploration is annealed from
-    # 1.0 to exploration_fraction over this number of timesteps scaled by
-    # exploration_fraction
-    "schedule_max_timesteps": 100000,
+    # === Exploration Settings ===
+    "exploration": {
+        "type": EpsilonGreedy,  # Exploration class.
```

Comment:
Can the type be a string instead of raw class? We probably should allow a class to be passed, but it would be better to have a string by default so it can be passed via YAML as well.

Reply:
Both would work.

Reply:
It gets constructed via `from_config`, which supports classes, strings, yaml/json-strings, and filenames.

Reply:
Ok, then let's make it String by default to avoid serialization problems. Users will also typically want to pass it in as a string.

Reply:
ok.